### PR TITLE
Demo title PR

### PR DESCRIPTION
**Pull request by Tushar01102000**
Fixed #3

### CODE REVIEW

**Comment by GangaChatrvedi** [comment]
file extension should be ```.json```

File: src/data/contributors/TUSHAR.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Tushar Kumar Jain",
+  "skills": ["C", "C++", "PYTHON", "WEB"],
+  "cause": "betterment",
+  "image": "Avatar5",
+  "github": "https://github.com/Tushar01102000",
+  "linkedin": "www.linkedin.com/in/tushar-kumar-jain-a34278198"
```
Comment (KeenWarrior) [inline src/data/contributors/TUSHAR.json:7]: add ```https://``` to the front of URL to make it ```https://www.linkedin.com/in/tushar-kumar-jain-a34278198```

**Comment by Abhishek-kumar09** [comment]
@Tushar01102000 ping!

**Comment by Abhishek-kumar09** [comment]
Welcome to @codeforcauseorg Tushar! Have a great journey!

**Pull request merged**